timescaledb/tutorials/grafana/visualizations/histograms.md

```diff
@@ -0,0 +1,200 @@
+<img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/multiple_histograms.png" alt="multiple histograms"/>
+We can clearly see the 3 distinct histograms but it's impossible to tell them apart from each
+other. Click on the green line left of the legend and pick a color.
+<img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/color_options.png" alt="color options"/>
+Doing so results in the following panel:
+<img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/colored_in_histogram.png" alt="colored in histogram"/>
+Here we can clearly see the 3 different price distributions for each stock.
+
+## Price/Volume Histogram
+An interesting metric to investigate is the trade volume distribution. This distribution gives us
+insight into the volume at which people buy a stock. The dataset has a column with the daily
+cumulative volume which we can use to determine the individual transaction volume.
+To calculate the volume of data for each bucket, we need to find the maximum `day_volume`
+value for the symbol and then subtract each value from the previous bucket maximum value.
+The difference of these two rows produces the volume within the bucket of time.
+We can accomplish this with a slightly more complex query that pre-aggregates our data using
+the TimescaleDB `time_bucket()` hyperfunction and the standard PostgreSQL `MAX()` function,
+and then use the PostgresSQL `LAG()` window function to subtract each row from the previous,
+ordered by `time` descending.
+```sql
+WITH buckets AS (
+    SELECT time_bucket('$bucket_interval', time) AS time,
+        symbol,
+        MAX(day_volume) AS dv_max
+    FROM stocks_real_time
+    WHERE time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+        AND day_volume IS NOT NULL
+        AND symbol in ($symbol)
+    GROUP BY time_bucket('$bucket_interval', time), symbol
+)
+SELECT TIME,
+    symbol,
+    CASE WHEN lag(dv_max ,1) OVER (PARTITION BY symbol ORDER BY time) IS    NULL THEN dv_max
+    WHEN (dv_max - lag(dv_max, 1) OVER (PARTITION BY symbol ORDER BY time)) < 0 THEN dv_max
+    ELSE (dv_max - lag(dv_max, 1) OVER (PARTITION BY symbol ORDER BY time)) END vol
+FROM buckets;
+```
+This query results in the following histogram:
+<img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/volume_distribution.png" alt="volume distribution"/>
+As you can see, we get a left-skewed distribution for the AMZN symbol. In many cases there will
+be outliers that represent a few very large volume transactions that distort the distribution. One
+solution to this problem is to only look at transactions with a volume less than a certain threshold.
+But that is outside the scope of this example.
```

Comment:
For now, let's either cut this sentence (just end it above) OR:

- Show an updated query with a filter based on `vol` (so, probably a having clause) - I don't think this means more screenshots. Just helpful info.
- Mention that one solution would be to use a logarithmic scale but unfortunately this chart doesn't yet support that.

Reply:
I used the latter in my latest commit ([2dade79](https://github.com/timescale/docs/pull/1081/commits/2dade79f64b405ee543babff11f2c816efebc096)).
Unless you meant for me to use both, then I can add another query with the filter 😄!